src/Commands/Reuniao.ts

```diff
@@ -0,0 +1,52 @@
+import Command from "@core/Contracts/Command"
+import Context from "@core/Contracts/Context"
+import InvalidArgsException from "@core/Exceptions/InvalidArgs"
+
+export default class Reuniao extends Command {
+  public get description() {
+    return "Coloca o servidor em modo reunião."
+  }
+
+  public get roles(): string[] {
+    return [process.env.ADMIN_ROLE!]
+  }
+
+  public get roleValidationMessages() {
+    return {
+      [process.env
+        .ADMIN_ROLE!]: "Apenas administradores podem usar esse comando"
+    }
+  }
+
+  public help(): string {
+    return ":x: Como usar: `!reuniao <on|off>`"
+  }
+
+  public validate({ arg }: Context): void | never {
+    const states = {
+      on: 1,
+      off: 2
+    }
+
+    if (!(arg in states)) {
```

Comment:
```suggestion
    // TODO: using in here it's a weird choice
    if (!(arg in states)) {
```

Reply:
@kjkGustavo is working on this command at the moment, so i'll leave it as it is for now